docs/reference/api/storage/BufferedBlockDevice.md

```diff
@@ -0,0 +1,23 @@
+## BufferedBlockDevice
+
+<span class="images">![](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/class_slicing_block_device.png)<span>BufferedBlockDevice class hierarchy</span></span>
```

Comment:
When the code merges, we'll need to update this link, as well as the class reference and example links.

Reply:
Sure. Had to push this PR first, as it was a requirement for the code PR to be merged.